### DISCUSSION TITLE
Branch creation date

### DISCUSSION THREAD

**Discussion by nricciardi** [gitpython-developers/GitPython#1596] [Q&A] [answered]
Is there any way to know the creation date of a branch?
I've already tried to get the commits of a branch and look for which one had the lowest date, but this doesn't work because if a branch has been generated from another branch, it will also have the commits of the original branch, thus returning an incorrect date (the one of the original branch).

**Comment by Byron** [comment, marked as answer]
Git does not keep track of branch creation dates. However, branches *may* just be files which do have a creation and/or modification date associated with them which might be useful as proxy in some situations. Note that branches can also be packed, which is when they are written into a single file, making it impossible to obtain timestamps as described here.
